src/utils/config.ts

```diff
@@ -2,4 +2,8 @@ import { config } from 'dotenv';
 
 config({ path: `${__dirname}/../../.env` });
 
+process.env.POSTGRES_URL = process.env.NODE_ENV === 'development'
+  ? process.env.DEV_POSTGRES_URL
+  : process.env.TEST_POSTGRES_URL;
+
 export default process.env;
```

Comment:
- There should be a `PORT` declared in `config.ts`
 - It's showing an error `[ERROR] 13:50:03 ⨯ Unable to compile TypeScript:
index.ts(4,13): error TS2304: Cannot find name 'PORT'.
`
- Not sure how it is still working as `nom run dev` starts the server on PORT 3000

Reply:
Agreed, I only declared the port in the .env file (but it will be gitignored).
The reason I didn't get the error, is because I had the port declared in the .env file.

Reply:
Can you tell me when does that error occur?
The reason why it's working is that, in config.ts, on line 9 we are exporting process.env.
In the index.ts file we import the config (which is basically an alias for process.env at this point). 
The first argument to the method listen called on app, is basically process.env.PORT.
Hope this clear everything up!

Reply:
When I run the server using `npm run dev`, that error is showing. But still the server is running. Can you please confirm if that error is showing up on your system as well or it just on-my-machine problem.

It's an interesting choice of making the module `config.ts`  reference the same object as `process.env`. I was under the impression (following FSO off-course) that the role of `config.ts` is to filter the environment variables that we need from `process.env`, add some extra environment variables and then making them available to the rest of the program as a separate module.  (Example - https://fullstackopen.com/en/part4/structure_of_backend_application_introduction_to_testing)

This makes `config.ts` the only place where all the environmental variables are declared, and also the only place where we use the object referenced by `process.env`.

Reply:
I know, I was also used to doing it as in FSO. This is Antonina's implementation, and I just built on top of it. Maybe she can clarify this?
The error does not happen at all on my machine.

Reply:
- I think we should discuss this with Antonina and make it confirm to the best practices.

Reply:
Ok, agreed!

Reply:
I agree with you both, for the sole reason of handling variable naming changes in one place (I don't see another issue rn).